src/v2/api/index.md

```diff
@@ -1272,100 +1272,100 @@ Todos los hooks de ciclo de vida tienen la instancia asignada automáticamente a
 
 - **Vea También:** [Diagrama de Ciclo de Vida](../guide/instance.html#Diagrama-de-ciclo-de-vida)
 
-## Directives
+## Directivas
 
 ### v-text
 
-- **Expects:** `string`
+- **Espera:** `string`
 
-- **Details:**
+- **Detalles:**
 
-  Updates the element's `textContent`. If you need to update the part of `textContent`, you should use `{% raw %}{{ Mustache }}{% endraw %}` interpolations.
+  Actualiza los elementos `textContent`. Si necesita actualizar la parte de `textContent`, debe usar interpolaciones `{% raw %}{{ Mustache }}{% endraw %}`.
 
-- **Example:**
+- **Ejemplo:**
 
   ```html
   <span v-text="msg"></span>
-  <!-- same as -->
+  <!-- igual que -->
   <span>{{msg}}</span>
   ```
 
-- **Vea también:** [Data Binding Syntax - interpolations](../guide/syntax.html#Text)
+- **Vea también:** [Sintaxis de vinculación de datos - interpolaciones](../guide/syntax.html#Text)
 
 ### v-html
 
-- **Expects:** `string`
+- **Espera:** `string`
 
-- **Details:**
+- **Detalles:**
 
-  Updates the element's `innerHTML`. **Note that the contents are inserted as plain HTML - they will not be compiled as Vue templates**. If you find yourself trying to compose templates using `v-html`, try to rethink the solution by using components instead.
+  Actualiza los elementos `innerHTML`. **Tenga en cuenta que los contenidos se insertan como HTML sin formato - no se compilarán como plantillas de Vue**. Si se encuentra intentando componer plantillas usando `v-html`, intente replantear la situación usando componentes en su lugar.
 
-  <p class="tip">Dynamically rendering arbitrary HTML on your website can be very dangerous because it can easily lead to [XSS attacks](https://en.wikipedia.org/wiki/Cross-site_scripting). Only use `v-html` on trusted content and **never** on user-provided content.</p>
+  <p class="tip">La representación dinámica de HTML arbitrario en su sitio web puede ser muy peligrosa porque puede conducir fácilmente a [ataques XSS](https://en.wikipedia.org/wiki/Cross-site_scripting). Solo use `v-html` en contenido de confianza y **nunca** en contenido proporcionado por el usuario.</p>
 
-- **Example:**
+- **Ejemplo:**
 
   ```html
   <div v-html="html"></div>
   ```
-- **Vea también:** [Data Binding Syntax - interpolations](../guide/syntax.html#Raw-HTML)
+- **Vea también:** [Sintaxis de vinculación de datos - interpolaciones](../guide/syntax.html#Raw-HTML)
 
 ### v-show
 
-- **Expects:** `any`
+- **Espera:** `any`
 
 - **Modo de Uso:**
 
-  Toggle's the element's `display` CSS property based on the truthy-ness of the expression value.
+  Alterna la propiedad `display` de CSS del elemento en función de la veracidad del valor de la expresión.
 
-  This directive triggers transitions when its condition changes.
+  Esta directiva desencadena transiciones cuando su condición cambia.
 
-- **Vea también:** [Conditional Rendering - v-show](../guide/conditional.html#v-show)
+- **Vea también:** [Rendering condicional - v-show](../guide/conditional.html#v-show)
```

Comment:
Me referia a estos terminos que podria traducirse como `Renderizado condicitional`. Hay bastantes mas de este estilo, pero no estoy seguro si Renderizado y rendering significa lo mismo.

Reply:
Yo creo que en el caso de Vue `render` es mas bien "traducir/interpretar". Renderizar se refiere (segun tengo entendido) al procesamiento de imagenes con un PC.

Aqui `render` es el proceso de traducir/interpretar codigo VUE en codigo HTML.

Quizas deberiamos votar, pero pienso que es la mejor opcion, ya que tambien al leer tiene mas sentido.

Reply:
voy a buscar, a ver si encuentro referencias

Reply:
Podemos dejarlo como `Renderización` yo lo he visto en la doc oficial de vuejs.org en otras traducciones como https://br.vuejs.org/v2/api/index.html#v-show . El termino `Renderizar` es uno de los más extendidos en español cuando hablamos de mostrar contenido (datos) desde un framework en el frontend, creo que ponerlo como `interpretar` o `traducir` generaría más confusión aún, así que la alternativa de `Renderización` me parece un término medio no tan malo.

Como queráis, cuando decidamos como lo dejamos subo cambios.

Reply:
Vale. Añadelo porfavor al Wiki y actualiza el PR. Avisame para que proceda con el merge ;)

Reply:
Hecho! :)